.circleci/config.yml

```diff
@@ -119,11 +119,8 @@ jobs:
           name: Run rubocop
           command: bundle exec rubocop
       - run:
-          name: Run erb_lint on Rails views
-          command: bundle exec erblint --lint-all
-      - run:
-          name: Run erb_lint on cells views
-          command: bundle exec erblint **/app/cells/**/*.erb
+          name: Run erblint
+          command: /app/.circleci/install_cc_test_reporter.sh
```

Comment:
You're installing the test reporter :smile:

Reply:
Damn copypaste